source/transactions/tests/write-concern.yml

```diff
@@ -244,8 +244,7 @@ tests:
           session: session0
           document:
             _id: 1
-        result:
-          insertedId: 1
+        # avoid assertion since result fields are inaccessible for unacknowledged write concern
```

Comment:
This command is part of a transaction so the driver ignores the w:0 write concern. A result *is* returned.